tensor2tensor/serving/query.py

```diff
@@ -85,21 +89,22 @@ def main(_):
       data_dir=os.path.expanduser(FLAGS.data_dir))
   problem.get_hparams(hparams)
   request_fn = make_request_fn()
-  while True:
-    inputs = FLAGS.inputs_once if FLAGS.inputs_once else input(">> ")
-    outputs = serving_utils.predict([inputs], problem, request_fn)
-    outputs, = outputs
-    output, score = outputs
-    print_str = """
-Input:
-{inputs}
-
-Output (Score {score:.3f}):
-{output}
-    """
-    print(print_str.format(inputs=inputs, output=output, score=score))
-    if FLAGS.inputs_once:
-      break
+
+  app = Flask(__name__.split(".")[0])
+
+  @app.route("/", methods=["GET"])
```

Comment:
This would be nice to have, but I don't think we can force everyone to use Flask. Could you change it so that Flask is just an option, maybe a parameter to the function that defaults to False (no Flask)?